src/containers/addNew.js

```diff
@@ -153,36 +64,90 @@ class AddNew extends Component {
 
   render() {
     return (
-      <div className="form">
-        <h1 className="form-label">Add a New Beer</h1>
-        <form onSubmit={this.onSubmit}>
-          {this.renderField('Beer name', 'beerName')}
-          {this.state.touched.beerName && this.state.beerName === '' ? (
-            <div className="error-message">Enter a beer name</div>
-          ) : null}
-
-          {this.renderNumberField('rating')}
-
-          {this.renderField('Description', 'desc')}
-          {this.state.touched.desc && this.state.desc === '' ? (
-            <div className="error-message">Enter a description</div>
-          ) : null}
-
-          {this.renderImgUpload('file')}
-          <div className="form-buttons">
-            <Link to="/" className="buttons back">
-              Cancel
-            </Link>
-            <button
-              disabled={this.state.isDisabled}
-              className="buttons submit"
-              type="submit"
-            >
-              Submit
-            </button>
+      <Formik
+        initialValues={initValues}
+        validate={values => {
+          let errors = {};
+          if (!values.beerName) errors.beerName = 'Enter a beer name';
+          if (!values.desc) errors.desc = 'Enter a description';
+          return errors;
+        }}
+        onSubmit={values => {
+          this.onSubmit(values);
+        }}
+      >
+        {({ errors, isValid, setFieldValue }) => (
+          <div className="form">
+            <Form>
+              <h1 className="form-label">Add a New Beer</h1>
+
+              {this.renderField('Beer name', 'beerName')}
+              {errors.beerName ? (
+                <div className="error-message">{errors.beerName}</div>
+              ) : null}
```

Comment:
You can use shortcut: `errors.beerName && <div className="error-message">{errors.beerName}</div>`